siac/walletcmd.go

```diff
@@ -61,7 +61,7 @@ By default the wallet encryption / unlock password is the same as the generated
 	}
 
 	walletLoad033xCmd = &cobra.Command{
-		Use:   "033x [filepath]",
+		Use:   "033x",
```

Comment:
the usage was chaged to exclude the filepath, definitely the correct option is to leave it off, and remove it from `siag` as well.